### ISSUE TITLE
Compatibility problem with ember-cli >= 0.2.3

### ISSUE THREAD

**Issue by DirkLachowski** [benefitcloud/ember-cli-uploader#2]
Doing  

```
npm install --save-dev ember-cli-uploader
ember generate ember-cli-uploader
```

got me

> Unknown blueprint: ember-cli-uploader

After that, i've changed

```
npm install --save-dev ember-cli-uploader
ember generate ember-cli-uploader
```

to 

```
ember install ember-cli-uploader
```

That throws an error:

> The `ember generate` command requires an entity name to be specified. For more details, use `ember help`.

I then manually ran 

```
ember generate ember-cli-uploader
```

and finally got a working addon. 

Maybe only a doc thing, but the warning regarding `ember generate` looks like some sort of a problem.
See also benefitcloud/ember-uploader#63

**Comment by natecox** [comment]
This is an issue with ember-cli 0.2.3 and is resolved now with 0.2.5.

**Comment by digitaltoad** [comment]
Thank you @natecox
